Source/FieldSolver/SpectralSolver/SpectralBinomialFilter.H

```diff
@@ -0,0 +1,41 @@
+#ifndef WARPX_SPECTRAL_BINOMIAL_FILTER_H_
+#define WARPX_SPECTRAL_BINOMIAL_FILTER_H_
+
+#include <HankelTransform.H>
+#include <SpectralKSpace.H>
+
+using namespace amrex;
```

Comment:
please do not use `using namespace` in public scope of header files, since this pulls all objects into the scope of all files that include this.
Instead, just add this inside `{}` scopes where needed.

Reply:
I cleaned this up in all of the files here.